test/serializer-test.ts

```diff
@@ -16,9 +17,11 @@ describe("serializer", function(): void {
     const value = "1000";
     const destination = "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh";
     const fee = "10";
+    const lastLedgerSequence = 20;
     const sequence = 1;
     const account = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ";
-    const publicKey = "testPublicKey";
+    const publicKey =
```

Comment:
This is a real public key. I just updated them so that I could perform real data validation.